docs/getting_started/quick_start.rst

```diff
@@ -97,7 +97,7 @@ With a read-only instance, you can do something like obtaining 10 "hot" submissi
 
     # continued from code above
 
-    for submission in reddit.subreddit("test").hot(limit=10):
+    for submission in reddit.subreddit("learnpython").hot(limit=10):
```

Comment:
Let's change r/learnpython to r/test in the sentence above to match the code sample. The change to r/test was deliberate.

Reply:
Ah, that makes sense. Ill make that change now